src/Keyboard.js

```diff
@@ -194,23 +197,26 @@ export default class Keyboard extends PureComponent {
 					{this.props.layouts.length > 1 ?
 						<KeyboardButton
 							value={<LanguageIcon />}
+							classes={'keyboard-languageButton'}
 							onClick={this.handleLanguageClick}
 						/>
 					: null}
 					{inputNode.dataset.type === 'email' ?
 						<KeyboardButton
 							value={'@'}
+							classes={'keyboard-atButton'}
 							onClick={this.handleLetterButtonClick}
 						/>
 					: null}
 					<KeyboardButton
 						value={' '}
-						classes="keyboard-space"
+						classes="keyboard-spaceButton"
 						onClick={this.handleLetterButtonClick}
 					/>
 					{inputNode.dataset.type === 'email' ?
 						<KeyboardButton
 							value={'.'}
+							classes={'keyboard-fullstopButton'}
```

Comment:
I haven't set eslint rules in this project, but it doesn't mean, that we shouldn't keep the code tidy. You pass `classes`-prop alternately by double quotes and curly braces. Can you please use the only way to do it?